easybuild/scripts/rpath_args.py

```diff
@@ -93,6 +93,8 @@
 
     # filter out --enable-new-dtags if it's used;
```

Comment:
actually, we already inject `--disable-new-dtags`, see line 114... so how can this patch fix anything?